### DISCUSSION TITLE
<Geojson /> is not capable to fill the whole google-map (the whole "earth")

### DISCUSSION THREAD

**Discussion by qobaid-bunnyrents** [react-native-maps/react-native-maps#4406] [Q&A]
### Summary

The whole google-maps (thus the whole earth) can be represented in a "Rectangle" shape with the following coordinates:

### [Lng , Lat]

- [-180, 90]   ---> left upper corner
- [180, 90]    ---> right upper corner
- [180, -90]   ---> right bottom corner
- [-180, -90]  ---> left bottom corner

Now if we want to render/draw a `<Geojson />` with only one Polygon (without holes/Interior-LinearRings) that has the previousely 4 mentioned coordinates, then nothing is really rendered/drawn by this `<Geojson />` component

### Reproducible sample code

```TSX
import React from 'react';
import MapView, {PROVIDER_GOOGLE, Geojson} from 'react-native-maps';

export default function App() {
  return (
    <MapView
      style={{flex: 1}}
      provider={PROVIDER_GOOGLE}
      initialRegion={{
        latitude: 42,
        longitude: 30,
        latitudeDelta: 1,
        longitudeDelta: 1,
      }}>
      <Geojson
        geojson={{
          type: 'FeatureCollection',
          features: [
            {
              type: 'Feature',

              geometry: {
                type: 'Polygon',
                coordinates: [
                  [
                    [-180, 90],
                    [180, 90],
                    [180, -90],
                    [-180, -90],
                    [-180, 90],
                  ],
                ],
              },
              properties: {},
            },
          ],
        }}
        fillColor="red"
      />
    </MapView>
  );
}
```

### Steps to reproduce

1. Render  a `<MapView />` component with "Google" as provider.
2. Use any initialRegion you want.
3. Render a `<Geojson />` component with only one Polygon  (without holes/Interior-LinearRings) that has the following coordinates for its Exterior-LinearRing: `[[-180, 90], [180, 90], [180, -90], [-180, -90], [-180, 90]]`

### Expected result

I expect to have a successfully rendered/drawn Polygon that covers the whole google-map/earth.

The following attachment shows a **correct/expected behavior**, rendered by **https://geojson.io**,  when using the same "geojson" provided in the "Reproducible sample code" section.

![Screen Shot 2022-08-18 at 11 41 32 PM](https://user-images.githubusercontent.com/109277113/185490878-197aaaa8-4b98-404a-a219-76a6bed5bbee.png)

### Actual result

Nothing is rendered/drawn by the `<Geojson />` component.

### React Native Maps Version

1.2.0

### What platforms are you seeing the problem on?

Android, iOS (Google Maps)

### React Native Version

0.67.4

### What version of Expo are you using?

Not using Expo

### Device(s)

iPhone 13 (iOS 15.0), Pixel 3 (Android 11)

### Additional information

Maybe this library tries to find the the shortest path when drawing a line between two points, and since the map is wrapable in the horizontal axis then the path's distance will be zero for all lines, thus nothing is drawn/rendered.

** Is there another way to achieve the expected behavior?

**Comment by onraj** [comment]
Did you find a fix? or is it a bug?

**Reply by qobaid-bunnyrents** [reply]
It's a bug, and unfortunately I couldn't find any workaround!

**Reply by onraj** [reply]
I found when you add more points it will work! 

Like this:
```json
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {},
      "geometry": {
        "type": "Polygon",
        "coordinates": [
          [
            [-180, -90],
            [-180, -60],
            [-180, -30],
            [-180, 0],
            [-180, 30],
            [-180, 60],
            [-180, 90],
            [-150, 90],
            [-120, 90],
            [-90, 90],
            [-60, 90],
            [-30, 90],
            [0, 90],
            [30, 90],
            [60, 90],
            [90, 90],
            [120, 90],
            [150, 90],
            [180, 90],
            [180, 60],
            [180, 30],
            [180, 0],
            [180, -30],
            [180, -60],
            [180, -90],
            [150, -90],
            [120, -90],
            [90, -90],
            [60, -90],
            [30, -90],
            [0, -90],
            [-30, -90],
            [-60, -90],
            [-90, -90],
            [-120, -90],
            [-150, -90],
            [-180, -90]
          ]
        ]
      }
    }
  ]
}

**Reply by qobaid-bunnyrents** [reply]
Thank you.

I will try it and will let you know.

**Reply by RemyNtshaykolo** [reply]
It worked thank you
